src/main/java/net/sf/jabref/exporter/SavePreferences.java

```diff
@@ -51,7 +51,7 @@ public static SavePreferences loadForExportFromPreferences(JabRefPreferences pre
     }
 
     public static SavePreferences loadForSaveFromPreferences(JabRefPreferences preferences) {
-        Boolean saveInOriginalOrder = true;
```

Comment:
I added a corresponding test. If `saveInOriginalOrder` is set to true, then even the save order specified in the metadata is ignored.